addons/mail/static/src/core/common/thread_model.js

```diff
@@ -98,7 +98,7 @@ export class Thread extends Record {
             return (
                 this.self_member_id?.is_pinned ||
```

Comment:
this one still reads `self_member_id` from thread, it's strange to have inconsistencies so close to each other